timbermill-java/timbermill-server/src/main/java/com/datorama/timbermill/server/service/TimbermillService.java

```diff
@@ -63,7 +63,7 @@ public TimbermillService(@Value("${INDEX_BULK_SIZE:200000}") Integer indexBulkSi
 			@Value("${DELETION_CRON_EXPRESSION:0 0 12 1/1 * ? *}") String deletionCronExp,
 			@Value("${DELETION_CRON_MAX_INDICES_IN_PARALLEL:1}") int expiredMaxIndicesToDeleteInParallel,
 			@Value("${DISK_HANDLER_STRATEGY:sqlite}") String diskHandlerStrategy,
-			@Value("${BULK_PERSISTENT_FETCH_CRON_EXPRESSION:0 0/10 * 1/1 * ? *}") String bulkPersistentFetchCronExp,
+			@Value("${BULK_PERSISTENT_FETCH_CRON_EXPRESSION:30 2/10 * 1/1 * ? *}") String bulkPersistentFetchCronExp,
 			@Value("${EVENTS_PERSISTENT_FETCH_CRON_EXPRESSION:0 0/5 * 1/1 * ? *}") String eventsPersistentFetchCronExp,
```

Comment:
Dvir asked me to change the "failed bulks" cron to not time-overlap the "overflowed events" cron, in order to check if is it the cause to the latest errors in sqlite:
![image](https://user-images.githubusercontent.com/54772016/100250333-3a769c00-2f46-11eb-94ce-178d7e057b36.png)

12:02:30,12:12:30,12:22:30,12:32:30,12:42:30
12:00:00,12:05:00,12:10:00,12:15:00,12:20:00

Reply:
nvm - saw you already changed it